data/events.json

```diff
@@ -0,0 +1,492 @@
+  },
+  {
+    "eventDbId": "6000000082",
+    "studyDbId": "6000000010",
+    "eventDescription": "mechanical harvest for green-chop biomass yield estimates, using Kemper small-plot harvester and customized Carter weigh-wagon",
+    "date": {
+      "startDate": "2017-09-11",
+      "endDate": "2017-09-16"
+    },
+    "eventParameters": [
+        {
+          "code": "HACOM",
+          "name": "harvest_operat_component",
+          "value": "aboveground biomass"
```

Comment:
ICASA says HACOM should be `"unit": "code"` but "aboveground biomass" is not one of the acceptable values. perhaps `"unit": "text"` for now until this new value can be added to ICASA

Reply:
going with

```
        {
          "code": "HACOM",
          "name": "harvest_operat_component",
          "unit": "code",
          "value": "canopy",
          "codeValueDescription": "entire aboveground biomass harvested, uncertain if 'canopy' is correct code"
        }
```

for now. Its not clear if 'canopy' has a more precise definition. But will follow up w/ ICASA folk.